sample_data/leica_1205_1.txt

```diff
@@ -1,2365 +0,0 @@
-                              System 1200 Data Export - File Begin
```

Comment:
Why delete this file?

Reply:
Cause it is the same file as this one https://github.com/steko/totalopenstation/blob/master/sample_data/leica_tcr_1205